### PR TITLE
Translations update from Hosted Weblate

### PR DESCRIPTION
**Pull request by weblate**
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [Jamulus/Jamulus app](https://hosted.weblate.org/projects/jamulus/jamulus-app/).

It also includes following components:

* [Jamulus/Windows Installer](https://hosted.weblate.org/projects/jamulus/windows-installer/)

Current translation status:

Weblate translation status

### CODE REVIEW

File: src/translation/translation_es_ES.ts [hunk 1]
```diff
@@ -4222,7 +4222,7 @@ Hemos silenciado tu canal y activado &apos;Silenciarme Yo&apos;. Por favor resue
     <message>
         <location filename="../util.h" line="653"/>
         <source>Server list full at directory</source>
```
[the remark this answers is not in the document]
Comment (pljones) [reply src/translation/translation_es_ES.ts:4224]: Capitalisation hasn't be done on the source yet.  Only renaming "directory server" to "directory".  There's _lots_ of capitalisation work that would need doing.

**Review by ann0see — COMMENTED** [review]
@jujudusud @trebmuh @dzpex @ignotus666 could you please review these translations? By first glance the structure is ok - but I can't say much on the translated content.

**Comment by trebmuh** [comment]
@ann0see I did the French translation myself, so all good from my point of view :)

**Comment by ignotus666** [comment]
Spanish is ok.

**Review by ignotus666 — APPROVED** [review]

**Comment by ignotus666** [comment]
@mapi68 @dzpex Looking at the Italian, I see that "Directory" is left as-is. Elswhere in the file, it's translated as "cartella", "indirizzo", "rubrica", "server centrale"... It needs a thorough review.

**Review by jujudusud — APPROVED** [review]
Thanks to @trebmuh, French part of the translation is reviewed and ok from my point.

File: src/translation/translation_pl_PL.ts [hunk 2]
```diff
@@ -154,7 +154,7 @@
     <message>
         <location filename="../aboutdlgbase.ui" line="72"/>
         <source>Copyright © 2005-2023 The Jamulus Development Team</source>
-        <translation type="unfinished"></translation>
+        <translation>Copyright (C) 2005-2023 Zespół programistów Jamulus-a</translation>
```
Comment (ann0see) [inline src/translation/translation_pl_PL.ts:157]: © would be better here. But I'm ok at the moment.

**Review by ann0see — APPROVED** [review]
I believe it's ok, but I can't say much about the actual content.

**Pull request merged**